### DISCUSSION TITLE
Atlantis template: incomplete figma for premium template

### DISCUSSION THREAD

**Discussion by arustamyansv** [primefaces/community#2431] [PrimeVue Templates] [answered]
Hello!

It seems there is an issue with Figma file that I partially mentioned in the #1879. 
I'd like to bring it up in the separate topic again cause our designer team requires Figma file for the work. 
Figma file doesn't contain components so we can design interface in Figma before implementation. 

If that's incorrect place to ask for such questions could you point me to where can I inquiry about my purchase please.

**Comment by onursenture** [comment, marked as answer]
Hello, component designs are not included in the scope of the template Figma files. We clearly share the scope of the Figma file you will obtain before purchasing via the Figma Preview link provided on the demo pages.

**Reply by arustamyansv** [reply]
Thanks for the reply,

Seems I indeed missed that Figma components are not included in the bundle...
Is there any way to purchase them? Is "UI Kit License" what I'm looking for?

**Reply by onursenture** [reply]
Yes, the UI Kit was designed to include all components. However, the themes we use in our Premium templates may differ from themes like Aura / Lara that we use in the open-source part of the library. For this reason, we recommend that you review the contents of the UI Kit before making a purchase.

**Reply by arustamyansv** [reply]
What about Atlantis components? 
Is there anything I can buy to work with them in Figma?
